dataland-frontend/src/components/resources/newLandingPage/AuthSection.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="header__authsection">
+    <a aria-label="Login to preview account" class="header__authsection-login" @click="login"> Login </a>
+    <button
+      aria-label="Sign up to preview account"
+      class="header__authsection-button"
+      name="signup_dataland_button"
+      @click="register"
+    >
```

Comment:
You could re-use the `RegisterButton` component here.

Reply:
Yes, I will use the same button component on every button

Reply:
Great 👍 . Will that happen in a different story?

If yes, let's use the RegisterButton here, just to merge a clean usage of that component to main, even if you will refactor button usage in a future Pull Request.

If no, I will leave this conversation open, until you tell me it is done. Then I will re-review this.